### PR TITLE
Add test for platform_util

### PR DESCRIPTION
**Pull request by stephanwlee**

### CODE REVIEW

File: tensorboard/util/platform_util_test.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+import tensorflow as tf
+
+from tensorboard.util import platform_util
+
+
+class PlatformUtilTest(tf.test.TestCase):
+
+  def test_readahead_file_path(self):
+    self.assertEqual('foo/bar', platform_util.readahead_file_path('foo/bar'))
```
Comment (wchargin) [inline tensorboard/util/platform_util_test.py]: Should this use `os.path.join`?

Comment (stephanwlee) [reply tensorboard/util/platform_util_test.py]: Nope, it is really weird function :)

**Comment by stephanwlee** [comment]
@wchargin hmm, what would it take to get LGTM?

**Comment by wchargin** [comment]
> Nope, it is really weird function :)

Okay, I’ll trust you on this—this is really surprising and should be
documented, but afaict this is conforming to some Google-internal API.

**Pull request merged**